web/source/kmwbase.ts

```diff
@@ -113,9 +113,9 @@ namespace com.keyman {
     getKeyboardPath(f, p?){return f;}
     KC_(n, ln, Pelem){return '';}
     handleRotationEvents(){}
-    // Will serve as an API function for a workaround, in case of future touch-alignment issues.
+    // Legacy API function for touch-alias issue workarounds
+    // Touch-aliases have been eliminated, though.
     ['alignInputs'](eleList?: HTMLElement[]){}
```

Comment:
Why are we keeping this here? Is it just because it was a published and documented API?

In this situation, we then need a PR against the documentation to mark it as deprecated, so it can be removed in a future version.

Reply:
https://help.keyman.com/DEVELOPER/ENGINE/WEB/15.0/reference/core/alignInputs